Configuration/Geometry/python/GeometryExtended2016Reco_cff.py

```diff
@@ -30,4 +30,5 @@
 from Geometry.CaloEventSetup.EcalTrigTowerConstituents_cfi import *
 from Geometry.EcalMapping.EcalMapping_cfi import *
 from Geometry.EcalMapping.EcalMappingRecord_cfi import *
+from Geometry.HcalCommonData.hcalDDConstants_cff import *
```

Comment:
@bsunanda - which PR pre seeds this one? I do not see this configuration fragment.